testing/upload_validation.ts

```diff
@@ -424,6 +424,7 @@ const defaultAuthenticationValidators: Record<AuthenticationType, z.ZodTypeAny>
     pkceChallengeMethod: z.enum(['plain', 'S256']).optional(),
     scopeParamName: z.string().optional(),
     nestedResponseKey: z.string().optional(),
+    credentialsLocation: z.string().optional(),
```

Comment:
Oops should have caught this earlier, this should be `z.nativeEnum(TokenExchangeCredentialsLocation).optional()`

Reply:
Fixed in https://github.com/coda/packs-sdk/pull/2354